.env.example

```
@@ -172,7 +172,7 @@ STORAGE_CIFS_DIR_MODE=0775
 # nobrl is deliberately absent: it disables byte-range locking, and Calibre and
 # Calibre-Web both write data/media/calibre-library/metadata.db, so SQLite
 # needs the server to arbitrate those locks.
-STORAGE_CIFS_EXTRA_OPTIONS=noperm,mfsymlinks,serverino,cache=strict,actimeo=30
+STORAGE_CIFS_EXTRA_OPTIONS=noperm,mfsymlinks,serverino,cache=strict,actimeo=120
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STORAGE_CIFS_EXTRA_OPTIONS key should go before the STORAGE_CIFS_FILE_MODE key

(UnorderedKey)